### PR TITLE
Document config flow for decora_wifi

### PR DESCRIPTION
**Pull request by TheBassEngineer**
Documentation change to go with [PR#51092](https://github.com/home-assistant/core/pull/51092).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/51092
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/decora_wifi.markdown [hunk 1]
```diff
@@ -17,24 +17,16 @@ Supported devices (tested):
...
+
+You will need your MyLeviton login information (username, which is usually your email address, and password) to use this module.
+
+## Configuration
+
+The preferred method for setting this integration up is by using the configuration flow. Go to the integrations page in your configuration and click on new integration -> Leviton Decora Wifi. Enter your credentials in the dialog and the integration will then set up. Once Home Assistant authenticates access, the `decora-wifi` integration will create the following platforms:
+
+- A `light` for each device linked to your MyLeviton account.
+
+The integration reads the capabilities of each device (dimming, etc.) from the API, so devices with dimming or speed control capabilities will provide those controls in the Home assitant UI.
```
Comment (frenck) [inline source/_integrations/decora_wifi.markdown]:
```suggestion

{% include integrations/config_flow.md %}

Once Home Assistant authenticates access, the Leviton Decora Wi-Fi integration will create the following:

- A `light` for each device linked to your MyLeviton account.

The integration reads the capabilities of each device (dimming, etc.) from the API, so devices with dimming or speed control capabilities will provide those controls in the Home Assitant UI.
```

File: source/_integrations/decora_wifi.markdown [hunk 2]
```diff
@@ -17,24 +17,16 @@ Supported devices (tested):
...
-username:
-  description: Your "My Leviton" app email address/user name.
-  required: true
-  type: string
-password:
-  description: Your "My Leviton" app password.
-  required: true
-  type: string
-{% endconfiguration %}
+## Setup
```
Comment (frenck) [inline source/_integrations/decora_wifi.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/decora_wifi.markdown [hunk 3]
```diff
@@ -17,24 +17,16 @@ Supported devices (tested):
...
+
+You will need your MyLeviton login information (username, which is usually your email address, and password) to use this module.
```
Comment (frenck) [inline source/_integrations/decora_wifi.markdown]:
```suggestion
You will need your MyLeviton login information (username, which is usually your email address, and password) to use this integration.
```

Comment (frenck) [reply source/_integrations/decora_wifi.markdown]: I wonder, is this set/prerequisites section really needed?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Additionally, this PR is targeting the `current` branch, but should be targeting the `next` branch. Please adjust the PR target branch and git rebase onto the `next` branch if needed.

Thanks 👍

File: source/_integrations/decora_wifi.markdown (hunk 3 above)
Comment (TheBassEngineer) [reply source/_integrations/decora_wifi.markdown]: Possibly not. I was following the format used by the doc page for Blink; it looks like other integrations don't put a heading over this portion of the doc.

**Comment by frenck** [comment]
You'll need to perform a git rebase onto the `next` branch in order to get rid of the unrelated commits in this PR.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by TheBassEngineer** [comment]
Closing this due to having closed the parent PR without merging.

**Pull request closed without merging**